lib/Dialect/SPIRV/Serialization/Serializer.cpp

```diff
@@ -131,6 +131,15 @@ class Serializer {
     return funcIDMap.lookup(fnName);
   }
 
+  uint32_t getOrCreateFunctionID(StringRef fnName) {
```

Comment:
Can you put a doc for this function? "// Gets the <id> for the function with the given name. Assigns the next available <id> if the function haven't been deserialized."